tests/test_ausschreibung.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime, timezone
+
+import pytest  # type:ignore[import]
+
+from bo4e.bo.ausschreibung import Ausschreibung, AusschreibungSchema
+from bo4e.bo.geschaeftspartner import Geschaeftspartner
+from bo4e.enum.ausschreibungsportal import Ausschreibungsportal
+from bo4e.enum.ausschreibungsstatus import Ausschreibungsstatus
+from bo4e.enum.ausschreibungstyp import Ausschreibungstyp
+from bo4e.enum.geschaeftspartnerrolle import Geschaeftspartnerrolle
+from tests.serialization_helper import assert_serialization_roundtrip  # type:ignore[import]
+from tests.test_adresse import example_adresse  # type:ignore[import]
+from tests.test_ausschreibungslos import example_ausschreibungslos  # type:ignore[import]
+from tests.test_zeitraum import example_zeitraum  # type:ignore[import]
+
+
+class TestAusschreibung:
+    @pytest.mark.parametrize(
+        "ausschreibung",
+        [
+            pytest.param(
+                Ausschreibung(
+                    ausschreibungsnummer="239230",
+                    ausschreibungstyp=Ausschreibungstyp.PRIVATRECHTLICH,
+                    ausschreibungsstatus=Ausschreibungsstatus.PHASE3,
+                    kostenpflichtig=True,
+                    ausschreibungportal=Ausschreibungsportal.BMWI,
+                    webseite="https://meineausschreibungswebsite.inv/",
+                    veroeffentlichungszeitpunkt=datetime(2022, 1, 1, 0, 0, 0, tzinfo=timezone.utc),
+                    abgabefrist=example_zeitraum,
+                    bindefrist=example_zeitraum,
+                    ausschreibender=Geschaeftspartner(
+                        name1="Batman",
+                        gewerbekennzeichnung=True,
+                        geschaeftspartnerrolle=[Geschaeftspartnerrolle.LIEFERANT],
+                        partneradresse=example_adresse,
+                    ),
+                    lose=[example_ausschreibungslos],
+                )
+            ),
```

Comment:
I would recommend to add a minimal example to this test.
Compare to [unittest in BO Rechnung](https://github.com/Hochfrequenz/BO4E-python/blob/0a48a815aef7c61c75856ca97e021ae6a570da10/tests/test_rechnung.py#L83)